scripts/generate_service.go

```diff
@@ -0,0 +1,305 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"go/format"
+	"log"
+	"os"
+	"strings"
+	"text/template"
+)
+
+type ComputeApiGenVersion int
+
+const (
+	none ComputeApiGenVersion = iota
+	v1
+	v0beta
```

Comment:
I think it may make sense to prefix these with `apiv`-- `apivNone` `apiv1`, `apiv0beta`, etc.--just to avoid claiming identifiers that may be shadowed within functions on accident.